component-studio/component-runtime-di/src/main/java/org/talend/sdk/component/runtime/di/record/DiRowStructVisitor.java

```diff
@@ -435,6 +435,8 @@ private void onArray(final Entry entry, final Collection value) {
     }
 
     private void onObject(final String name, final Object value) {
+        //in studio di, the object value is suitable to json string always? this code for construct the input tck record to tck processor, maybe is ok
+        //will adjust it if meet real requirement, then not toJson, only call "with(name, value)" api;
```

Comment:
should call with(schema.getEntry("name"), value) api

Reply:
jsonb may help us a lot for ser/deser. See my comment on slack.